api/db/changelog/db.changelog-103-data-dictionary.xml

```diff
@@ -0,0 +1,43 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+  xmlns="http://www.liquibase.org/xml/ns/dbchangelog/1.9"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog/1.9
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-1.9.xsd">
+  <changeSet author="ericsong" id="changelog-103-data-dictionary">
+
+    <createTable tableName="data_dictionary_entry">
+      <column name="data_dictionary_entry_id" type="bigint" autoIncrement="true">
```

Comment:
opt: I realize we have mixed conventions in the codebase, but WDYT about using `id` as the primary id column name as a convention?

Reply:
i think I have a slight preference for the `tableName_id` convention as it reduces the likelihood of errors in writing join queries.